website/src/css/_footer.scss

```diff
@@ -1,13 +1,78 @@
+$footer-background-color: #FDFDFD;
+$footer-border-top-color: rgba(19, 20, 25, 0.15);
+$footer-shadow-color: rgba(0, 0, 0, 0.25);
+
 .footer {
-  background-color: #ffffff;
-  border-top: 1px solid #f2eeeee8;
+  background-color: $footer-background-color;
+  /* no global variable */
+  border-top: 1px solid $footer-border-top-color;
+  @include t2;
 }
 
 [data-theme='dark'] .footer {
-  background-color: #12161b;
-  border-top: 1px solid #202429;
+  background-color: var(--detox-color-a1, #12161B);
+  box-shadow: 0px 4px 4px 0px $footer-shadow-color;
+  border-top: 1px solid var(--detox-color-a3, #202429);
+}
+
+@media only screen and (max-width: 996px) {
+  .footer__col {
+    margin-bottom: calc(var(--ifm-spacing-vertical) * 2.8);
+  }
+
+  .footer__col:last-child .footer__title {
+    margin-bottom: 7px;
+  }
+
+  .footer__link-item a {
+    margin-bottom: 12px;
+  }
+}
+
+.footer__title {
+  @include t2b;
+}
+
+
+.socialNetLinks {
+
+  .footer__link-item {
+    display: inline;
+    padding-left: 30px;
+    background-repeat: no-repeat;
+    gap: 0;
+  }
+}
+
+@media only screen and (min-width: 320px) and (max-width: 996px) {
+  .socialNetLinks {
+    display: flex;
+    justify-content: space-between;
+    flex-wrap: wrap;
+    max-width: 375px;
+  }
+}
+
+.github {
```

Comment:
The class names are not isolated well.

Use .footer__link-item_github, ... and so on